content/news/2025-XX-XX-bevy-0.16/index.md

```diff
@@ -29,8 +32,14 @@ Since our last release a few months ago we've added a _ton_ of new features, bug
 The features above may be great, but what else does Bevy have in flight?
 Peering deep into the mists of time (predictions are _extra_ hard when your team is almost all volunteers!), we can see some exciting work taking shape:
 
-- TODO
+- **A revamped observers API:** Observers are incredibly popular, but come with some weird quirks. Why is `OnAdd` not generic? What does that `B: Bundle` generic even _do_? Please, I want to add observers in my entity spawning to write little callbacks!
+- **Resources-as-entities:** Sure would be nice if hooks, observers, relations and more worked with resources. Rather than duplicating all of the code, we'd like to [make them components on singleton entities](https://github.com/bevyengine/bevy/pull/17485) under the hood.
+- **A .bsn file format and bsn! macro:** With the foundations laid (required components, improved spawning and relations!), it's time to build out the terse and robust Bevy-native scene format (and matching macro) described in [bevy#14437](https://github.com/bevyengine/bevy/discussions/14437).
+- **Light textures:** Also known as "light cookies", these are great for everything from dappled sunlight to shadow boxing.
+- **Deep Learning Super Sampling:** DLSS is a modern, GPU and neural-net powered approach to improving the resolution of images: coming soon to an open source game engine near you.
```

Comment:
I think we should make it clear that we're integrating _nvidia's_ DLSS, and not writing our own :P

Also it's not really "improving the resolution of images". It's basically better TAA than what we have, and because it's an upscaler, makes the frame overall much quicker to render (rendering at a lower resolution saves a lot of GPU time, even if the upscaling step itself is (likely?) more expensive than bevy's TAA).

Reply:
```suggestion
- **NVIDIA Deep Learning Super Sampling:** DLSS is a modern, GPU and neural-net powered approach to upscaling,. With `wgpu`'s help, we're hoping to integrate with NVIDIA's built-in solution.
```

Reply:
Suggested something like this:
-  **NVIDIA Deep Learning Super Sampling:** DLSS is a neural-net powered approach to temporal anti-aliasing and upscaling for NVIDIA RTX GPUs. We're working on integrating DLSS into Bevy to provide a cheaper and higher quality anti-aliasing solution than Bevy's current TAA (on supported platforms).


The wgpu work was also stuff I did btw 😅. They weren't involved in the actual DLSS/Bevy part, but they helped me write and debug a new wgpu API that allowed me to integrate DLSS.

Reply:
Awesome thank you (both for the writing and for the upstream work!).
```suggestion
**NVIDIA Deep Learning Super Sampling:** DLSS is a neural-net powered approach to temporal anti-aliasing and upscaling for NVIDIA RTX GPUs. We're working on integrating DLSS into Bevy to provide a cheaper and higher quality anti-aliasing solution than Bevy's current TAA (on supported platforms).
```